changelog.txt

```diff
@@ -30,6 +31,7 @@ new features:
 -Trigger times now include the time with respect to the first interaction (vertex times)
 -Analog to digital converter module added
 -Improved calculation of the diode noise parameters
+-Added the ability to force polarization to be only eTheta or ePhi in the reconstruction of the electric field using voltageToEfieldConverter.py
```

Comment:
I think this also needs to go up to the most recent version.

Reply:
Yeah I wasn't sure if the dates represent start times or end times for that particular version. But I had moved it to version 1.1.2